28jan2021 seen xml2po word not exist in the work dir of x11-terms/xfce4-terminal-0.8.10 on gentoo!

apr 17 2020: on archlinux
diff --git a/configure.ac.in b/configure.ac.in
index 9d683e6..0e79fd6 100644
--- a/configure.ac.in
+++ b/configure.ac.in
@@ -113,10 +113,10 @@ if test x"$enable_gen_doc" = x"yes"; then
         enable_gen_doc=no
     fi
 
-    AC_PATH_PROG([XML2PO], [xml2po], [no])
-    if test x"$XML2PO" = x"no"; then
-        enable_gen_doc=no
-    fi
+#    AC_PATH_PROG([XML2PO], [xml2po], [no])
+#    if test x"$XML2PO" = x"no"; then
+#        enable_gen_doc=no
+#    fi
 
     dnl we checked for xmllint above
     if test x"$XMLLINT" = x"no"; then
